charts/astronomer/templates/houston/houston-deployment.yaml

```diff
@@ -37,6 +37,18 @@ spec:
       {{- if and (not .Values.houston.airflowBackendSecretName) (not .Values.houston.airflowBackendConnection) }}
       serviceAccountName: {{ template "houston.bootstrapperServiceAccount" . }}
       initContainers:
+        - name: houston-wait-for-db
+          image: {{ template "houston.image" . }}
+          imagePullPolicy: {{ .Values.images.houston.pullPolicy }}
+          command:
+            - "/houston/bin/entrypoint"
+          args:
+            - "echo"
+            - "DB ready"
+          resources:
+{{ toYaml .Values.houston.resources | indent 12 }}
+          env:
+            {{- include "houston_environment" . | indent 12 }}
```

Comment:
@sjmiller609 @verdverm so this is going to run this script - https://github.com/astronomer/houston-api/blob/master/bin/entrypoint, then print "DB Ready", right?

This same script actually runs before Houston itself boots up by I guess that's fine. Having it in the initContainer should also prevent Houston from hanging, while waiting to run migrations or mid-migration, leaving it in a locked state.

cc @pritt20

Reply:
yes exactly

Reply:
the reason is so that it runs before db-bootstrapper does.